### ISSUE TITLE
Displacement of extensor tendon seen on imaging

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9875]
**Preferred term label:**
Displacement of extensor tendon seen on imaging

**Synonyms**
Dislocation of tendon, subluxation of tendon

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sagittal band disruption

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
which tendon?

**Comment by MickeySegal** [comment]
So far we've used it for https://www.orthobullets.com/hand/6029/sagittal-band-rupture where they listed tendons

- EDC/EIP/EDM
- lumbricals
- interossei

It might make most sense to add the word "hand" to the finding name.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000788
- label: Displacement of the extensor tendon in metacarpophalangeal joint
- parents: [Abnormal finger morphology (HP:0001167)](https://hpo.jax.org/app/browse/term/HP:0001167)
- pmid: PMID:37425263
- def: Dislocation of the extensor tendons from their central position on the metacarpal heads can be traumatic, congenital, spontaneous, or degenerative.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9875
- synonyms:
